umm-spec-lib/resources/example-data/umm-json/variable/v1.2/V1200267323-HMR_TME_v_1_2.json

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
There's a lot of inconsistent indentation in this example file.

Reply:
Yeah ... I just pasted 'em as I got 'em ... they're small; I can fix them up.

Reply:
Done.